source/clear-linux/tutorials/greengrass.rst

```diff
@@ -1,40 +1,42 @@
 .. _greengrass:
 
-Enable AWS Greengrass* and OpenVINO™ on |CL-ATTR|
-#################################################
-
-Hardware accelerated Function-as-a-Service (FaaS) enables cloud developers
-to deploy inference functionalities [1] on Intel® IoT edge devices with
-accelerators (CPU, Integrated GPU, Intel® FPGA, and Intel® Movidius™). These
-functions provide a great developer experience and seamless migration of
-visual analytics from cloud to edge in a secure manner using a containerized
-environment. Hardware-accelerated FaaS provides the best-in-class
+Enable AWS Greengrass\* and OpenVINO™ on |CL-ATTR|
```

Comment:
Simplify page title to not use "Clear Linux OS" e.g. "Enable AWS Greengrass\* and OpenVINO™"